src/main/java/com/snowflake/kafka/connector/internal/streaming/TopicPartitionChannel.java

```diff
@@ -1034,6 +1040,7 @@ private SnowflakeStreamingIngestChannel openChannelForTable() {
             .build();
     LOGGER.info(
         "Opening a channel with name:{} for table name:{}", this.channelName, this.tableName);
+    this.channelTryOpenCount.incrementAndGet();
```

Comment:
Sorry I dont understand use of this metrics, this will always be 1 right? 

Right place would be to increment in `openChannelForTable` right?